data_architect_misc/data_transformer/transform_functions/latam_dominican_republic_transform_functions.py

```diff
@@ -0,0 +1,68 @@
+"""This is the subclass of Transform function for Dominican Republic (LATAM division).
+
+We will define transform functions specific to Dominican Republic here.
+
+Author: Maicol Contreras
+Last Modified: September 9, 2020
+"""
+
+import pandas as pd
+import os,re
+from glob import glob
+
+from constants import comp_harm_constants
+from constants.transform_constants import KEY_CURRENT_INPUT_FILE, KEY_DELIMITER, KEY_HEADER
+from transform_functions.common_comp_harm_transform_functions import CommonCompHarmTransformFunctions
+
+
+class LatamDominicanRepublicTransformFunctions(CommonCompHarmTransformFunctions):
+    """
+    All custom (uncommon) transform functions **SPECIFIC to
+    individual processing task** must be defined as part
+    of this class.
+    """
+    DOMINICAN_REPUBLIC_SPECIFIC_CATEGORY_MAPPINGS = {
+        "(?i).*TRAPEADOR.*": "Home Care",
+    }
+
+    def __init__(self, config):
+        self.config = config
+
+    def add_RAW_MEDIA_column_for_multiple_files_using_specific_string_part_of_filename(
```

Comment:
If we are extracting raw media type names from the file names, what happens if someone incorrectly name the file (and the media type is incorrectly named)? How do we prevent that? Should we throw an exception here? Or will we let the QA config file catch it (try naming a file incorrectly for media type and see if the QA process passes it correctly)?

Reply:
Don Plartoo, it's much better if we will let the QA config file catch it the media wrong